File: pset0/mario.c
// Imports libraries and establishes main class.
#include <stdio.h>
#include <cs50.h>

int main()
{
    // Gets height input.
    int height = get_int("Height: ");

    // Tests input validity.
    while (height > 8 || height < 1)
    {
        height = get_int("Height: ");
    }

    // Creates for loop to make structure.
    for (int i = 0; i < height; i++)
    {
        // Prints spaces.
        for (int h = (height - (height - (i + 1))); h < height; h++)
        {
            printf(" ");
        }

        // Print hashes.
        for (int j = (height -  i); j < height; j++)
        {
            printf("#");
        }

        // Prints final hash.
        printf("#\n");
    }

}

File: pset6/mario.py
from cs50 import get_int

MAXHEIGHT = 8
MINHEIGHT = 1


# Takes input and rerequests it if need be.
height = get_int("Height: ")

# Casts height as int and checks to see if viable.
while MAXHEIGHT < height or height < MINHEIGHT:
    height = get_int("Height: ")


# Creates Pyramid.
for i in range(1, height + 1):

    # Prints beginning spaces. 
    for j in range(height-i):
        print(" ", end='')

# Prints left hashes.
    for k in range(height-(height-i)):
        print("#", end='')

# Prints center.
    print("  ", end='')

# Prints right hashes.
    for m in range(height-(height-i)):
        print("#", end='')
    
    print("")
    
